src/components/header.jsx

```diff
@@ -105,10 +105,14 @@ function Header({ slug }) {
       <Banner slug={slug}>{t('header.banner')}</Banner>
       <div className="usa-navbar">
         <div className="header__logo-title">
-          <img className="header__logo" src={ctoLogoShortform} alt="NYC CTO" />
+          <img
+            className="header__logo"
+            src={ctoLogoShortform}
```

Comment:
Minor adjustment: `ctoLogoShortForm`

Reply:
I was trying to be consistent with the names of the logos from the mocto website which are like `cto_logo_shortform_dark` and are treating shortform and longform as 1 word

Reply:
Should we change the file and variable names so that it is more generalized for agencies?

Reply:
I'm updating the variable names now. I was thinking maybe we can have a file like `src/index.js` that exports all of the images, and that would be the one place where the image paths need to change based on the file names

Reply:
Great idea, go for it!